src/scaladoc/scala/tools/nsc/doc/model/MemberLookup.scala

```diff
@@ -100,11 +100,36 @@ trait MemberLookup extends base.MemberLookupBase {
   private def isJDK(sym: Symbol) =
     sym.associatedFile.underlyingSource.map(f => isChildOf(f, (sys.props("java.home")))).getOrElse(false)
 
-  def jdkUrl(path: String): String = {
-    if (path.endsWith(".jmod")) {
+  // ISSUE-12820
```

Comment:
the new code beginning here is incorrectly indented, mind correcting that?

Reply:
Passed the new code into scalafmt like `pbpaste | scalafmt --stdin --stdout` to help me find and auto fix errors. Used the following `.scalafmt.conf`.
```
version = 3.8.4-RC1
runner.dialect = scala213
```

Reply:
I'm sorry, I was wrong